src/app/clusters/time-format-localization-server/app_config_dependent_sources.cmake

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2025 Project CHIP Authors
+# Copyright (c) 2021-2025 Project CHIP Authors
```

Comment:
I think this was not created in the past - the codegen sources was from this year.